Makefile

```diff
@@ -96,21 +96,21 @@ kind-cluster: ## Standup a kind cluster for e2e testing usage
 
 ##@ install/run:
 
-install-apis: cert-mgr generate kustomize ## Install the core rukpak CRDs
-	$(KUSTOMIZE) build manifests/apis | kubectl apply -f -
+install-apis: generate ## Install the core rukpak CRDs
+	$(KUSTOMIZE) build manifests/apis/crds | kubectl apply -f -
```

Comment:
I think that the core webhook is a part of the api.

Reply:
I agree the core webhook can be part of the default installation but since it's a webhook and not a CRD should we treat it differently?

Reply:
I think that the CRDs and it's validation should installed together.  WDYT?

Reply:
I think that's fair -- I'm looking for a way to do that and not duplicate the cert-manager installation though. Maybe the way to do this is to check whether cert-manager is installed before installing it?

Reply:
> I think that the CRDs and it's validation should installed together. WDYT?

I'm sure where I land with this tbh. I think my initial impression is that a user running `make install-apis` would result in only the CRDs being stamped out on a cluster, but at the same time, I can understand the perspective that we should also be installed the validation webhook. I don't have a strong opinion though: my current preference would be to separate them out into individual targets, and it looks like Dan's already doing this throughout these changes.

Reply:
I had originally combined them because there's really no case where we want the CRDs to exist without the webhooks. If they could, that would allow custom objects to be created and stored in etcd that violate the checks that the webhook performs and thus potentially violate assumptions we make in the controllers.

In fact, I was on the path of combining _all_ of the rukpak repo's manifests into a single `kustomize build ./manifests | kubectl apply -f` invocation.